docs/guide/coordinates.rst

```diff
@@ -0,0 +1,103 @@
+    (49.84856512, 0.05394699, 1417743.94689472)>
+
+
+Observer Location
+-----------------
+
+Both `~sunpy.coordinates.frames.Helioprojective` and `~sunpy.coordinates.frames.Heliocentric` frames are defined based on the position of the observer.
+Therefore to transform either of these frames to a different frame the location of the observer must be known.
+The observer can be specified for a coordinate object using the ``observer`` argument to `~astropy.coordinates.SkyCoord`.
+For sunpy to calculate the location of Earth or another solar-system body, it must know the time associated with the coordinate; this is specified with the ``obstime`` argument.
+
+Using the observer location it is possible to convert a coordinate as seen by one observer to a coordinate seen by another::
+
+  >>> hpc1 = SkyCoord(0*u.arcsec, 0*u.arcsec, observer="earth",
+  ...                 obstime="2017-07-26",
+  ...                 frame=frames.Helioprojective)
+
+  >>> hpc1.transform_to(frames.Helioprojective(observer="venus",
+  ...                                          obstime="2017-07-26"))
+  <SkyCoord (Helioprojective: obstime=2017-07-26T00:00:00.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate for 'venus'>): (Tx, Ty, distance) in (arcsec, arcsec, AU)
+      (-1285.47497992, 106.20918654, 0.72405937)>
+
+
+Using Coordinates with Maps
+---------------------------
+
+.. plot::
+   :include-source:
+
+   sunpy Map uses coordinates to specify locations on the image, and to plot overlays on plots of maps.
+   When a Map is created, a coordinate frame is constructed from the header information.
+   This can be accessed using ``.coordinate_frame``:
+
+   >>> import sunpy.map
+   >>> from sunpy.data.sample import AIA_171_IMAGE   # doctest: +REMOTE_DATA
+   >>> m = sunpy.map.Map(AIA_171_IMAGE)  # doctest: +REMOTE_DATA
+   >>> m.coordinate_frame  # doctest: +REMOTE_DATA
+   <Helioprojective Frame (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+       (-0.00406308, 0.04787238, 1.51846026e+11)>)>
+
+   This can be used when creating a `~astropy.coordinates.SkyCoord` object to set the coordinate system to that image:
+
+   >>> from astropy.coordinates import SkyCoord
+   >>> import astropy.units as u
+   >>> c = SkyCoord(100 * u.arcsec, 10*u.arcsec, frame=m.coordinate_frame)  # doctest: +REMOTE_DATA
+   >>> c  # doctest: +REMOTE_DATA
+   <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+       (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+       (100., 10.)>
+
+   The `~astropy.coordinates.SkyCoord` object can be converted to a PixelPair object using `~sunpy.map.GenericMap.world_to_pixel`:
```

Comment:
```suggestion
   The `~astropy.coordinates.SkyCoord` object can be converted to a `.PixelPair` object using `~sunpy.map.GenericMap.world_to_pixel`:
```
Links it hopefully

Reply:
I don't think this will work because `PixelPair` isn't in the API docs: https://docs.sunpy.org/en/stable/search.html?q=pixelpair&check_keywords=yes&area=default